eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -75,23 +76,42 @@ public RegionWSServerV2() {
     @GetMapping(value = "/{regionId}/variants")
     @ResponseBody
     @RateLimit(value = REGION_REQUEST_RATE_LIMIT)
-    public ResponseEntity getVariantsByRegion(@PathVariable("regionId") String regionId,
-                                              @RequestParam(name = "species") String species,
-                                              @RequestParam(name = "assembly") String assembly,
-                                              @RequestParam(name = "studies", required = false) List<String> studies,
-                                              @RequestParam(name = "annot-ct", required = false) List<String>
-                                                      consequenceType,
-                                              @RequestParam(name = "maf", required = false) String maf,
-                                              @RequestParam(name = "polyphen", required = false) String polyphenScore,
-                                              @RequestParam(name = "sift", required = false) String siftScore,
-                                              @RequestParam(name = "annot-vep-version", required = false) String
-                                                      annotationVepVersion,
-                                              @RequestParam(name = "annot-vep-cache-version", required = false) String
-                                                      annotationVepCacheVersion,
-                                              @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                              @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                              HttpServletResponse response,
-                                              @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByRegion(
+            @ApiParam(value = "Comma separated values of regions, in format chr:start-end. e.g. 13:32884600-32884700")
+            @PathVariable("regionId") String regionId,
+            @ApiParam(value = "First letter of the genus, followed by the full species name, e.g. hsapiens. " +
```

Comment:
I suggest that "hsapiens" and "grch37" are made examples for illustration purposes of a default search. Same applied to numbers. Basically the same comment as in `GeneWSServerV2`.